local/http/http.go

```diff
@@ -51,13 +52,25 @@ type Server struct {
 	AllowHTTP     bool
 	Logger        zerolog.Logger
 	Appversion    string
+	UseGzip       bool
 
 	httpserver  *http.Server
 	httpsserver *http.Server
 
 	serverPort string
 }
 
+var gzipContentTypes = []string{
+	"text/html",
+	"text/plain",
+	"text/csv",
+	"text/javascript",
+	"text/css",
+	"application/json",
+	"application/javascript",
+	"application/vnd.api+json",
+}
+
```

Comment:
If we want to restrict the gzipped content type I think we should at least add the following:
- text/xml 
- image/svg+xml

and even maybe those ones:
- application/atom+xml
- application/rss+xml